python/lsst/meas/algorithms/objectSizeStarSelector.py

```diff
@@ -317,18 +339,15 @@ def selectStars(self, exposure, sourceCat, matches=None):
         plotMagSize = lsstDebug.Info(__name__).plotMagSize             # display the magnitude-size relation
         dumpData = lsstDebug.Info(__name__).dumpData                   # dump data to pickle file?
 
-        # create a log for my application
-        logger = pexLogging.Log(pexLogging.getDefaultLog(), "meas.algorithms.objectSizeStarSelector")
-
         detector = exposure.getDetector()
         pixToTanXYTransform = None
         if detector is not None:
-            tanSys = detector.makeCameraSys(cameraGeom.TAN_PIXELS)
+            tanSys = detector.makeCameraSys(TAN_PIXELS)
             pixToTanXYTransform = detector.getTransformMap().get(tanSys)
```

Comment:
I'm going to ask you to file a separate ticket because I don't understand what part of it would be of interest to other star selectors.